src/commands/aksReconcileCluster/aksReconcileCluster.ts

```diff
@@ -0,0 +1,39 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { reconcileUsingUpdateInCluster, determineProvisioningState, getAksClusterTreeItem } from '../utils/clusters';
```

Comment:
I think determineProvisioningState needs to be removed from the imports as well.